app/app.js

```diff
@@ -9,12 +10,38 @@ app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true}));
 app.use(express.static(path.join(__dirname, '..', 'public')));
 
-/*
-=====
-OTHER ROUTES GO HERE
-=====
-*/
+app.get('/api/formdata', function(req, res) {  // Will return all entries
```

Comment:
set this route to -> app.get('/api/entries'....)

/api/entries -> returns the users entries

/api/formconfig -> with a type parameter -> returns the users form config / options that appear in their forms